satpy/dataset/metadata.py

```diff
@@ -33,7 +33,8 @@ def combine_metadata(*metadata_objects, average_times=True):
     in them and consisting of datetime objects will be averaged. This
     is to handle cases where data were observed at almost the same time
     but not exactly.  In the interest of time, lazy arrays are compared by
-    object identity rather than by their contents.
+    object identity rather than by their contents. Raw dataset metadata
+    are excluded as well.
```

Comment:
Would you mind putting the actual key here, like `('raw_metadata')`.

Reply:
Sure!